src/lib/Service/Service.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\SystemInfo\Service;
+
+/**
+ * @internal
+ */
+interface Service
```

Comment:
I'd rather name it as:
```suggestion
interface ServiceInfo
```